lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,47 +2,21 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_include_tree
+
         def initialize(serializer, options = {})
           super
-          @include_tree = IncludeTree.from_include_args(options[:include] || '*')
+          self._include_tree = IncludeTree.from_include_args(options[:include] || '*')
         end
 
-        def serializable_hash(options = nil)
+        def serializable_hash(options = {})
```

Comment:
It's currently not consistent with the JsonApi adapter, plus it's outside the scope of this PR.